snowplow-common-internal/pubsub/src/main/scala/com.snowplowanalytics.snowplow/sources/PubsubSource.scala

```diff
@@ -94,7 +96,7 @@ object PubsubSource {
           LowLevelEvents(events, acks)
         }
         .evalTap { case LowLevelEvents(events, _) =>
-          val numBytes = events.map(_.size).sum
+          val numBytes = events.map(_.limit()).sum
```

Comment:
Just a warning, I fixed a bug on this line in the main branch: https://github.com/snowplow-incubator/snowplow-lake-loader/commit/95fab3c39507f69c23ff551b07564fded6946632

So we'll need to reconcile these two changes before we publish this as a libray.

One option is to rebase everything now onto `main`.  So we reconcile it before we do the copy&paste into the other repo.  WDYT?

Reply:
I agree, after code review is completed, I'll rebase this branch onto `main`